.cirrus.yml

```diff
@@ -66,10 +66,10 @@ linux_qa_task:
   <<: *ONLY_SONARSOURCE_QA
   gke_container:
     dockerfile: it/docker/Dockerfile
-    builder_image_project: ci-cd-215716
+    builder_image_project: sonarqube-team
```

Comment:
Out of curiosity: is this a brand new GCE Project? Or you renamed the existing "ci-cd-215716" one? If the former, I guess we will move all other projects, too? If the latter... I guess everything would start breaking down, so it seems unlikely 😅 .

Reply:
It is a brand new one.
I will move every project which belongs to the SQ team to new GCP project.